### PR TITLE
Remove flag from language dropdown

### LINKED ISSUE #5245: Remove flags from language dropdown in header
Flags represent a country, not a language. Showing a flag for a language is incorrect. In some cases these match, in some cases these don't. The most obvious example is showing the UK flag for English.

image

Other websites handle this correctly and don't display flags for languages, for example

LinkedIn:

image

Google translate:

image

Facebook:

image

### PR DESCRIPTION
**Pull request by Goosly**
fixes #5245  (maybe, I've not yet succeeded in getting the project running locally so I can't test)

### CODE REVIEW

File: WcaOnRails/app/views/translations/index.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
       <div class="panel-heading <%= class_heading %>" data-toggle="collapse" data-target="#collapse-<%= locale %>">
         <h3 class="panel-title">
           <span class="label label-<%= class_panel %>">i18n</span>
-          <span class="label label-<%= class_panel %>"><%= flag_icon Locales::AVAILABLE[locale][:flag_id] %> <%= locale %></span> <%# this is all on one line for a reason. See http://stackoverflow.com/a/25879479 %>
+          <span class="label label-<%= class_panel %>"><%= locale %></span> <%# this is all on one line for a reason. See http://stackoverflow.com/a/25879479 %>
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/translations/index.html.erb]: Pretty sure the SO link is no longer relevant here =)

**Comment by viroulep** [comment]
>I've not yet succeeded in getting the project running locally

Are you running into some specific issues?

**Comment by viroulep** [comment]
I updated the PR to include the "language" icon from font awesome, and also fixed some visibility issues that were occurring at lower screen sizes.
I'll merge when tests pass.

**Comment by viroulep** [comment]
Merged and deployed, but I'm still interested in knowing if you had any issue with getting the website to work locally.

**Comment by Goosly** [comment]
I've not yet succeeded in getting the dependencies installed. Currently I'm stuck on:

`An error occurred while installing therubyracer (0.12.3), and Bundler cannot     continue.                                                                        Make sure that `gem install therubyracer -v '0.12.3' --source                    'https://rubygems.org/'` succeeds before bundling.`

Anyway, not sure if anyone has ever succeeded in getting this project running on Windows :-)

**Comment by viroulep** [comment]
We removed that gem a while ago, @Ivan-Brigidano got the same issue on OSX and the gem wasn't actually used anymore!
I'm pretty sure no one tried much getting this to run on windows (and I'm not sure we want to invest a lot of time to make sure it does, since it runs on a linux server).
But if it's just small errors like this it's for sure stuff that we can figure out.

**Comment by Ivan-Brigidano** [comment]
I was able to solved it installing some needed dependences before installing therubyfacer (libv8 I think). Anyways, as Philippe said, this gem has been removed, so it should be easier to update your fork from the orinal repo and the run `bundle install`.

**Pull request merged**